### PR TITLE
chore: Update example tests

### PR DESCRIPTION
**Pull request by vdmgolub**

### CODE REVIEW

File: scripts/generateExamplesTests.js [hunk 1]
```diff
@@ -90,18 +90,20 @@ function fixupExamples(examples) {
           const goldenFilePath = `./testdata/${jobKey}_comment_golden.md`;
           const commentArgs = step.run
             .replace(/\\/g, '')
+            .replace(/--pull-request \$\{\{github\.event\.pull_request\.number\}\}/g, '--pull-request 1')
```
Comment (hugorut) [inline scripts/generateExamplesTests.js:93]: why are we changing this to a constant?

**Review by hugorut — COMMENTED** [review]
minor query

File: scripts/generateExamplesTests.js (hunk 1 above)
Comment (vdmgolub) [reply scripts/generateExamplesTests.js:93]: It also runs on master, the PR number is empty, but the value is required, thus it throws an error. I think we still want to run it on master, and having the #1 should be enough to check the API fetch.

**Review by hugorut — APPROVED** [review]
LGTM

**Pull request merged**